packages/web/src/store/wallet/sagas.ts

```diff
@@ -40,6 +46,10 @@ const errors = {
   rateLimitError: 'Please wait before trying again'
 }
 
+const ASSOCIATED_TOKEN_ACCOUNT_CREATION_FEE = new BN(
```

Comment:
nit: Technically the amount of SOL is slightly higher than this, but this should 80/20 it. If we want to be precise, there's an RPC method for getting the minimum rent for exemption given a byte size. I use it in the But Audio flow

Reply:
Nice!